react/react1/week2/my-app/src/components/TodoList.js

```diff
@@ -0,0 +1,68 @@
+import { useState } from "react";
+import todoData from "../data/TodoData";
+
+
+export default function TodoList() {
+    const [todoState, setTodoState] = useState(todoData);
+
+    const AddNewTodo = () => {
+        setTodoState((prevTodos) => {
+            const newTodo = {
+                id: prevTodos[prevTodos.length - 1].id + 1,
+                description: "Random text",
+                status: false,
+            };
+            return [...prevTodos, newTodo];
+        });
+    };
+
+    function TaskDescription(props) {
+        return (
+            <span>
+                {props.description}
+            </span>
+        )
+    };
+
+    const handleOnChange = (id) => {
```

Comment:
Maybe use a better name here? handleOnChange is a bit vague, for instance, onTodoChecked